<launch>
  <node name="carla_sensor_publisher_node" pkg="carla_sensor_publisher" type="carla_sensor_publisher_node" output="screen">
    <param name="sensor_definition_file" default="$(find agv)/config/sensors.json"/>
    <param name="sensor_ns" default="sensors/"/>
  </node>
</launch>